transport/server.cc

```diff
@@ -1027,7 +1020,7 @@ cql_server::connection::process(uint16_t stream, request_reader in, service::cli
     if (bounce_msg) {
         auto shard = (*bounce_msg)->move_to_shard().value();
         auto&& cached_vals = (*bounce_msg)->take_cached_pk_function_calls();
-        co_return co_await process_on_shard(shard, stream, is, client_state, trace_state, dialect, std::move(cached_vals), process_fn);
+        msg = co_await process_on_shard(shard, stream, is, client_state, trace_state, dialect, std::move(cached_vals), process_fn);
```

Comment:
This should still be co_return, no? The while isn't there yet.

Reply:
I guess it doesn't matter since we'll return it below anyway.

Reply:
Yes, it's returned 2 lines later so I thought it's better to have less `return`s  from the function, also this reduces the number of changes in the next commit.